conftest.py

```diff
@@ -170,6 +170,8 @@ def _collect_item_metadata(item):
             result["testDeclaration"] = "flaky"
         elif result["details"].startswith("bug"):
             result["testDeclaration"] = "bug"
+        elif result["details"].startswith("incomplete_test_app"):
+            result["testDeclaration"] = "incompleteTestApp"
```

Comment:
@colin-higgins can we update the feature parity dashboard to support this new test declaration? 

Also it would be nice if tests marked with incompleteTestApp show up with a color similar to missing feature (maybe light grey and dark grey?)